### PR TITLE
Omics setup

### PR DESCRIPTION
**Pull request by rosemm**
Wrapper module for learners to launch the AMI that can then be used for subsequent practical omics modules.

### CODE REVIEW

**Review by leemc-data-ed — CHANGES_REQUESTED** [review]
Just a couple of small things, but I think this is generally good to go!

File: genomics_setup/genomics_setup.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+<!--
+
+author:   Rose Hartman
+email:    hartmanr1@chop.edu
+version:  0.0.1
+current_version_description: Initial version
+module_type: wrapper
+docs_version: 1.1.0
```
[the remark this answers is not in the document]
Comment (rosemm) [reply genomics_setup/genomics_setup.md:8]: Good catch! Thank you!

**Comment by rosemm** [comment]
Thanks, @leemc-data-ed! I think this is all set now.

File: genomics_setup/genomics_setup.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
+<!--
+
+author:   Rose Hartman
+email:    hartmanr1@chop.edu
+version:  0.0.1
```
Comment (leemc-data-ed) [inline genomics_setup/genomics_setup.md:5]:
```suggestion
version:  1.0.0
```

**Review by leemc-data-ed — CHANGES_REQUESTED** [review]
I think this is the last thing, then it should be good to go!

**Comment by rosemm** [comment]
Okay, @leemc-data-ed I think this is ready!

**Review by leemc-data-ed — APPROVED** [review]
Lgtm!

**Pull request merged**